zerver/migrations/0328_migrate_to_edit_topic_policy.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.2 on 2021-05-26 09:43
+
+from django.db import migrations
+from django.db.backends.postgresql.schema import DatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+
+
+def migrate_to_edit_topic_policy(apps: StateApps, schema_editor: DatabaseSchemaEditor) -> None:
+    Realm = apps.get_model("zerver", "Realm")
+    Realm.POLICY_EVERYONE = 5
+    Realm.POLICY_ADMINS_ONLY = 2
+    Realm.objects.filter(allow_community_topic_editing=False).update(
+        invite_to_realm_policy=Realm.POLICY_ADMINS_ONLY
+    )
+    Realm.objects.filter(allow_community_topic_editing=True).update(
+        invite_to_realm_policy=Realm.POLICY_EVERYONE
```

Comment:
This should be `edit_topic_policy`.  I caught this by manually testing the migration -- setting each configuration and running the migration back and forth to confirm it worked as expected in the UI.  I highly recommend doing that sort of testing when writing migrations, mainly because bugs in them corrupt data and thus are much more expensive than other bugs.  

(Also, it's another reason to write a reverse migration, which I did while testing this).

Reply:
Ah, this was a big mistake, will be more careful now onwards and will make sure to the manual testing thoroughly.